src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py

```diff
@@ -52,10 +52,10 @@ def _sorting(self, obj, reverse_order=False):
                 obj.values(),
                 key=lambda b: b.price / b.energy))
 
-    def _discrete_point_curve(self, obj_list):
+    def _discrete_point_curve(self, obj_list, functor):
```

Comment:
It was my bad naming this to something as generic as functor, I would rename this to something like round_functor

Reply:
Yes, will change that, thanks!